### PR TITLE
Classes

### PR DESCRIPTION
**Pull request by lackeyjb**
{
  "comfortability": 3,
  "completeness": 5,
  "questions": nil"
}

### CODE REVIEW

**Comment by lackeyjb** [comment]
@rgpass @drmikeh

File: talkative_robot.rb [hunk 1]
```diff
@@ -1,85 +1,144 @@
...
+	def pickup_item?(input)
+		gets_answer_about_groceries == "Y" || gets_answer_about_groceries == "yes"
+	end
+
+	def grocery_list_styling
+		grocery_list.map.with_index { |item, index| "#{index + 1} -- #{item}" }
+	end
+
+	def owner
+		user
```
Comment (rgpass) [inline talkative_robot.rb]: `@user`

File: talkative_robot.rb [hunk 2]
```diff
@@ -1,85 +1,144 @@
...
+		@user = user 
+	end
+
+	def gets_answer_about_groceries
+		puts "Hey! did you get the #{grocery_list}? (Y/N)"
+		gets.chomp
+	end
+
+	def random_grocery_item
+		random_item = grocery_list.sample
```
Comment (rgpass) [inline talkative_robot.rb]: All of these should be instance variables. `@grocery_list`

File: talkative_robot.rb [hunk 3]
```diff
@@ -1,85 +1,144 @@
...
+	def to_s
+		"you need to pick up the following from the store: #{grocery_list}."
+	end
 end
 
-user = { name: get_users_name, age: get_users_age, gender: get_users_gender }
+name = User.get_users_name
+age = User.get_users_age
+gender = User.get_users_gender
+fav_spot = User.get_users_favorite_place
```
Comment (rgpass) [inline talkative_robot.rb]: Align `=` signs.

File: talkative_robot.rb [hunk 4]
```diff
@@ -1,85 +1,144 @@
...
+current_user = User.new(name: name, age: age, gender: gender, fav_spot: fav_spot)
+
+current_user.prints_user_message
+current_user.user_turns_75_message
+current_user.prints_user_age_message
+
+bryan_the_author = Author.new({name: "Bryan", age: 25, gender: gender, fav_spot: "Yellowstone"})
```
Comment (rgpass) [inline talkative_robot.rb]: Should just be able to do `Author.new` and have everything hard coded.

File: talkative_robot.rb [hunk 5]
```diff
@@ -1,85 +1,144 @@
...
+puts bryan_the_author.to_s
+
+user = User.new ({ name: "Brittany", age: "23", gender: "F" })
```
Comment (rgpass) [inline talkative_robot.rb]: No need to do this twice. Already created a `current_user`

**Comment by rgpass** [comment]
Change all the local variables to instance variables and you'll be good to go.

**Comment by lackeyjb** [comment]
@rgpass

**Comment by rgpass** [comment]
:+1:

**Pull request merged**